abs.yaml

```diff
@@ -0,0 +1,4 @@
+upload_channels:
+  - boldorider4-test-channel
+channels:
+  - boldorider4-test-channel
```

Comment:
Please delete abs.yaml before merging.

Reply:
I sure will. I should have specified that I will in the description.